libraries/rdiff_backup.rb

```diff
@@ -71,7 +71,7 @@ class RdiffBackup < LWRPBase
           end
         end
       end
-      action :create do
+      action :create do # rubocop:disable BlockLength
```

Comment:
This breaks the jenkins linting test because its using an older version of rubocop. You can either leave this here but use two pound signs so rubocop can skip it, or just remove it for now.